phys2denoise/cli/run.py

```diff
@@ -0,0 +1,196 @@
+                             'bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rrf', '--rrf',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rrf',
+                        help='Respiratory response function. Needs the following inputs: '
+                             'sr, os, tl, onset and tr.',
+                        default=False)
+    metric.add_argument('-rcard', '--retroicor-card',
+                        dest='metric_list',
+                        action='append_const',
+                        const='r_card',
+                        help='Computes regressors for cardiac signal. Needs the following '
+                             'inputs: tr, nscans, slt and n_harm.',
+                        default=False)
+    metric.add_argument('-rresp', '--retroicor-resp',
+                        dest='metric_list',
+                        action='append_const',
+                        const='r_resp',
+                        help='Computes regressors for respiratory signal. Needs the following  '
+                             'inputs: tr, nscans, slt and n_harm.',
+                        default=False)
+    optional.add_argument('-outdir', '--output-dir',
+                          dest='outdir',
+                          type=str,
+                          help='Folder where output should be placed. '
+                               'Default is current folder.',
+                          default='.')
+    optional.add_argument('-sr', '--sample-rate',
+                          dest='sample_rate',
+                          type=float,
+                          help='Sampling rate of the physiological data in Hz.',
+                          default=None)
+    optional.add_argument('-pk', '--peaks',
+                          dest='peaks',
+                          type=str,
+                          help='Filename of the list with the indexed peaks\' positions'
+                               ' of the physiological data.',
+                          default=None)
+    optional.add_argument('-thr', '--throughts',
+                          dest='throughts',
+                          type=str,
+                          help='Filename of the list with the indexed peaks\' positions'
+                               ' of the physiological data.',
+                          default=None)
+    optional.add_argument('-os', '--oversampling',
+                          dest='oversampling',
+                          type=int,
+                          help='Temporal oversampling factor in seconds. '
+                               'Default is 50.',
+                          default=50)
+    optional.add_argument('-tl', '--time-length',
+                          dest='time_length',
+                          type=int,
+                          help='RRF Kernel length in seconds.',
```

Comment:
Does this apply to the CRF kernel as well?

Reply:
I think so, but as I said I based these inputs in PR #4